### ISSUE TITLE
All links need accessibility scanners criteria

### ISSUE THREAD

**Issue by joelworsham** [JacksonCollege/jc-theme-2015#232]
They use a pseudo element for the visual representation of hover/focus, but this doesn't meet criteria for typical accessibility scanners. I need to add something to the links themselves that the scanner can pick up on. Perhaps a subtle color change.

**Comment by joelworsham** [comment]
I added a non-visible background change on hover for links that were not meeting this criteria; the "animated" links that have the bar underneath. Can you see if this little "hack" makes them pass the scan? (changes will be on live site within an hr)

**Comment by banksashleym** [comment]
Hi Joel, sorry, we were closed wed-fri last week for spring break. 

Everything seems to pass except for the link in the actual button? When I have the "future students" tab open and run the scanner, it is saying `<button> 5` doesn't pass...and this occurs when I switch tabs and run the scanner again. I'm not sure if this is the link within the button or the actual button itself?

Thanks for your help!

![capture](https://user-images.githubusercontent.com/15264315/37595766-5c7244a0-2b50-11e8-85db-03893fbd8a27.JPG)

**Comment by joelworsham** [comment]
Hmm interesting. Does it mention why it doesn't pass? What criteria it isn't meeting?

**Comment by banksashleym** [comment]
I take it back...the error is coming from the slick slider buttons!

`<button type="button" role="tab" id="slick-slide-control02" aria-controls="slick-slide02" aria-label="3 of 6" tabindex="-1" style="z-index: 2147483646; position: relative;">3</button>`

Apparently these need a focus visible as well? Sorry about the confusion!

**Comment by joelworsham** [comment]
I think we covered this in another issue, right? Can you do me a favor and check this again? Thanks!

**Comment by banksashleym** [comment]
Yes, these pass now :)
